user-manual/collection.md

```diff
@@ -0,0 +1,37 @@
+# Collection
+
+## Favorite Albums
+
+You can add or delete albums to the list by clicking the star button.
+
+![Add Favorite Albums](https://i.imgur.com/xVp5iFO.png)
```

Comment:
Thanks for taking the screenshots, could you please make sure they're in their original size, so they're not blurry? And please crop them to the exact size of the window so there are no white borders.

Reply:
I tried to modify it according to your feedback. The pixels in the current picture are 1385 x 918. If the picture is still blurry or broken, can you tell me the ideal pixel?

Reply:
Thanks, it looks great now. Would you like to add anything else, or is it ready to merge?

Reply:
I want to work more!